packages/mobile/src/screens/edit-track-screen/screens/AdvancedOptionsScreen.tsx

```diff
@@ -21,14 +26,12 @@ export const AdvancedOptionsScreen = () => {
       bottomSection={null}
       variant='white'
     >
-      <View>
-        <SubmenuList>
-          <IsrcField />
-          <LicenseTypeField />
-          {/* Only show key and bpm field if we are in edit flow */}
-          {trackId ? <KeyBpmField /> : <></>}
-        </SubmenuList>
-      </View>
+      <SubmenuList>
+        <IsrcField />
+        <LicenseTypeField />
+        {isUnlisted ? null : <ReleaseDateField />}
+        {isUpload ? <></> : <KeyBpmField />}
```

Comment:
nbd, but any reason to return fragment instead of null?

Reply:
it has do do with the "List" component above, v awkward, we should update that component